tasks/fine_tuning_sbert/src/sentence_transformer.py

```diff
@@ -199,42 +212,37 @@ def fit(self,
                 print(f'Epoch: {epoch}')
                 print(f"Best score: {self.best_score}")
                 print('=' * 60)
-                self.plot_train_val_accuracy(model_deets, output_path, training_acc_list)
                 return
 
             # removing the unnecessary first element in ACC_LIST that needed to be there for epoch 1
             if epoch == 0:
                 del self.acc_list[0]
 
-        if evaluator is None and output_path is not None:  # No evaluator, but output path: save final model version
+        # No evaluator, but output path: save final model version
+        if evaluator is None and output_path is not None:
             self.save(output_path)
 
-    def plot_train_val_accuracy(self, model_deets, output_path, training_acc_list):
-        plt.plot(range(1, len(training_acc_list) + 1), training_acc_list, 'orange', label='Training accuracy')
-        plt.plot(range(1, len(self.acc_list) + 1), self.acc_list, 'blue', label='Validation accuracy')
-        plt.title('Training and Validation accuracy')
-        plt.xlabel('Epochs')
-        plt.ylabel('Accuracy')
-        plt.rcParams["figure.figsize"] = [15, 15]
-        plt.legend()
-        plt.savefig(
-            f"{output_path}/{model_deets}.png",
-            bbox_inches="tight", )
-        plt.show()
-
     def _eval_during_training(self, evaluator, output_path,
                               model_deets, save_best_model, epoch, steps, callback):
         """Runs evaluation during the training"""
 
-        score_dict = evaluator(self, output_path=output_path, model_deets=model_deets, epoch=epoch, steps=steps)
+        score_dict = evaluator(self, output_path=output_path,
+                               model_deets=model_deets, epoch=epoch, steps=steps)
 
         score = score_dict["accuracy"]
         self.acc_list.append(score)
 
+        wandb.log({"validation_acc": score, "epoch": epoch})
+        wandb.log(
+            {"Macro F1 validation": score_dict['macro_f1'], "epoch": epoch})
+        wandb.log(
+            {"Weighted F1 validation": score_dict['weighted_f1'], "epoch": epoch})
+
         prev_score = self.acc_list[-2]
         moving_average = mean(self.acc_list[-self.patience - 1: -1])
 
-        print(f"{'=' * 60}\nCurrent Score is: {score}\nCurrent ACC_LIST is: {self.acc_list}")
+        print(
```

Comment:
Is there a particular reason why you didn't choose to log this with wandb? Just curious :)

Reply:
The validation accuracy values are stored in wandb one epoch at a time individually :) the ``ACC_LIST`` is only for monitoring the training in the notebook while it happens.
Do you think we should remove this now since the accuracy is stored in wandb? 🤔